hiworkload/compile.sh

```diff
@@ -1,8 +1,20 @@
 #!/bin/sh
 # SPDX-License-Identifier: GPL-2.0-or-later
-# Copyright (c) 2019 Petr Vorel <pvorel@suse.cz>
+# Copyright (c) 2019-2022 Petr Vorel <pvorel@suse.cz>
```

Comment:
A nit but this is generally not required and 2019 would still be fine here. JFYI.